os/kernel/binary_manager/binary_manager.h

```diff
@@ -118,20 +112,31 @@ struct faultmsg_s {
 typedef struct faultmsg_s faultmsg_t;
 #endif
 
-/* Binary data type in binary table */
-struct binmgr_bininfo_s {
+/* User binary data type in binary table */
+struct binmgr_uinfo_s {
```

Comment:
I am not sure but if you use binary as a file in file system, you can get kernel version anytime without reveal user binary. (open - read kernel binary file)
is char kernel_ver[] necessary?